### PR TITLE
Add LED BLE integration

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Integrates multiple brands of Bluetooth Low Energy LED devices into Home Assistant.
These devices have been sold under many brands, including:

- LED BLE
- BLE-LED
- Triones
- Happy Lighting
- Zengee
- LEDBlue
- Zerproc

I plan on adding these as supported brands in the future once I have a few more of the devices in hand. ... Aliexpress shipping is slow

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/77489
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3620
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/led_ble.markdown [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: LED BLE
+description: Instructions on how to integrate LED BLE devices into Home Assistant.
+ha_category:
+  - Light
+ha_bluetooth: true
+ha_release: 2022.9
+ha_iot_class: Local Push
```
Comment (bdraco) [inline source/_integrations/led_ble.markdown]:
```suggestion
ha_iot_class: Local Polling
```

File: source/_integrations/led_ble.markdown [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- LEDBlue
+
+These devices have been sold under many brands, including:
+
+- LED BLE
+- BLE-LED
+- Triones
+- Happy Lighting
+- Zengee
+- LEDBlue
```
Comment (bdraco) [inline source/_integrations/led_ble.markdown:37]:
```suggestion
- LEDBlue
- Zerproc
```

File: source/_integrations/led_ble.markdown [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+This integration works with devices that identify with the following names:
+
+- LEDnet (except LEDnetWF models)
+- BLE-LED
+- LEDBLE
+- Triones
+- LEDBlue
+
+These devices have been sold under many brands, including:
```
Comment (bdraco) [inline source/_integrations/led_ble.markdown:30]: I plan on adding these as supported brands in the future once I have a few more of the devices in hand.  ... Aliexpress shipping is slow

**Review by frenck — APPROVED** [review]
Thanks, @bdraco 👍

**Comment by bdraco** [comment]
Thanks

**Pull request merged**